lib/placeholders/card/readMe.md

```diff
@@ -0,0 +1,35 @@
+## Navbar
```

Comment:
Please fix the title.

Reply:
977c4770419b2a5b4289c7e62a8812e8b16cf095